client/extensions/woocommerce/app/settings/payments/payment-method-list.js

```diff
@@ -76,7 +86,9 @@ class SettingsPaymentsMethodList extends Component {
 function mapStateToProps( state, ownProps ) {
 	const paymentMethods = getPaymentMethodsGroup( state, ownProps.methodType );
 	const site = getSelectedSiteWithFallback( state );
+	const isLoading = ! arePaymentMethodsLoaded( state );
```

Comment:
unfortunately payment methods are not loading till there is a site id which is usually most of the wait so you only see the placeholder for a very short period after waiting without them.